<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addDialog</class>
 <widget class="QDialog" name="addDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QPushButton" name="previous_pushButton">
     <property name="text">
      <string>Previous</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="dateLabel">
       <property name="text">
        <string>Date</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="date_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="timeLabel">
       <property name="text">
        <string>Time</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="time_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Callsign </string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="callsignLineEdit">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>rx RST</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>tx RST</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="rxrst_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>59</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>58</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>56</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>54</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>53</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>51</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="txrst_comboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
       <item>
        <property name="text">
         <string>59</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>58</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>57</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>56</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>55</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>54</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>53</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>52</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>51</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="name_lineEdit"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>QTH</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="qth_lineEdit"/>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="next_pushButton">
     <property name="text">
      <string>Next</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="reset_pushButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="cancel_pushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QPushButton" name="add_pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3" colspan="2">
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Contact No.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="contact_lineEdit">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="modeComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frequency</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="freqLineEdit">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Country</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>State/Region</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>County</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="countryComboBox"/>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="stateComboBox">
       <item>
        <property name="text">
         <string>MO</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="countyComboBox_6">
       <item>
        <property name="text">
         <string>Boone</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
    </layout>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="currentRecord">
     <property name="text">
      <string>Record</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>callsignLineEdit</tabstop>
  <tabstop>rxrst_comboBox</tabstop>
  <tabstop>txrst_comboBox</tabstop>
  <tabstop>name_lineEdit</tabstop>
  <tabstop>qth_lineEdit</tabstop>
  <tabstop>add_pushButton</tabstop>
  <tabstop>cancel_pushButton</tabstop>
  <tabstop>reset_pushButton</tabstop>
  <tabstop>previous_pushButton</tabstop>
  <tabstop>next_pushButton</tabstop>
  <tabstop>date_lineEdit</tabstop>
  <tabstop>time_lineEdit</tabstop>
  <tabstop>stateComboBox</tabstop>
  <tabstop>countyComboBox_6</tabstop>
  <tabstop>contact_lineEdit</tabstop>
  <tabstop>countryComboBox</tabstop>
  <tabstop>modeComboBox</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>freqLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
